ecommerce-ui/src/RentalSelector.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class RentalSelector extends Component {
+    render() {
+        const availableListings = this.props.listings.map((listing, idx) => {
+            return (
+                <div key={idx} className="listing-each">

```

Comment:
Would like to see these divs Refactored into modular components! Often in React anytime you use a map and it starts to fill with JSX its a good indication to create a reusable component.